zzz_generated/all-clusters-app/zap-generated/access.h

```diff
@@ -0,0 +1,359 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+// THIS FILE IS GENERATED BY ZAP
+
+// Prevent multiple inclusion
+#pragma once
+
+// Prevent changing generated format
+// clang-format off
+
+#define GENERATED_ACCESS_PRIVILEGE__VIEW (0)
+#define GENERATED_ACCESS_PRIVILEGE__OPERATE (1)
+#define GENERATED_ACCESS_PRIVILEGE__MANAGE (2)
+#define GENERATED_ACCESS_PRIVILEGE__ADMINISTER (3)
+
+////////////////////////////////////////////////////////////////////////////////
+
+// Parallel array data (*cluster*, attribute, privilege) for read attribute
+#define GENERATED_ACCESS_READ_ATTRIBUTE__CLUSTER { \
+    31, /* Cluster: Access Control, Attribute: ACL, Privilege: administer */ \
+    31, /* Cluster: Access Control, Attribute: Extension, Privilege: administer */ \
+    /* Cluster: Door Lock, Attribute: DoorOpenEvents, Privilege: view */ \
```

Comment:
It's a little confusing that things that don't list access at all don't show up here (and use the default) whereas those that list the default show up (and also use the default).

I would vote we only list the non-default things here, and don't even mention the ones that are default, no matter how they got there.

Reply:
I can go either way on this, I guess...